tests/molecule/test_graphs.py

```diff
@@ -0,0 +1,28 @@
+import sys
+sys.path.append('/data/yuanqi/graphein')
+
+from graphein.molecule.graphs import construct_graph
+from graphein.molecule.config import MoleculeGraphConfig
+
+config = MoleculeGraphConfig()
+
+def test_generate_graph_sdf():
```

Comment:
We need some more checks on the graphs. We can follow a similar model to the protein graph tests.